packages/playground/src/components/contracts_list/contracts_table.vue

```diff
@@ -89,6 +89,38 @@
     </template>
   </weblet-layout>
 
+  <v-dialog width="800" v-model="contractStateDialog">
+    <v-card>
+      <v-card-title class="text-h5 mt-2"> Contract lock Details</v-card-title>
+      <v-card-text>
+        <v-row class="d-flex justify-center">
+          Amount Locked:
+          {{ contractLocked?.amountLocked ?? 0 > 0 ? contractLocked?.amountLocked.toFixed(3) : 0 }}
+          TFTs.
+        </v-row>
+
+        <v-alert v-if="contractLocked?.amountLocked == 0" class="ma-4" type="warning" variant="tonal"
```

Comment:
can we be more specific with this condition? we can check if the user has a rent contract on this node or not

Reply:
should be fixed.